libexec/bats-core/bats

```diff
@@ -116,6 +121,10 @@ report_formatter=''
 recursive=
 export BATS_TEMPDIR_CLEANUP=1
 output=
+export TRACE_LEVEL="${TRACE_LEVEL:-}"
```

Comment:
Why not BATS_TRACE_LEVEL? Also, I'd rather "firewall" the value to not transfer via the ENV.

Reply:
> Also, I'd rather "firewall" the value to not transfer via the ENV.

Sorry, can you clarify what you mean?

I can change the ENV names 👍🏼

Reply:
When running the selftest suite setting BATS_TRACE_LEVEL during the outer bats run would result in enabling it for the inner run too. I'd like to avoid that.

Reply:
What do you recommend? I don't think I quite understand to be able to change what you want me to :( Sorry